packages/block-library/src/latest-posts/index.php

```diff
@@ -14,38 +14,37 @@
  */
 function render_block_core_latest_posts( $attributes ) {
 	$args = array(
-		'numberposts' => $attributes['postsToShow'],
-		'post_status' => 'publish',
-		'order'       => $attributes['order'],
-		'orderby'     => $attributes['orderBy'],
+		'posts_per_page'   => $attributes['postsToShow'],
+		'post_status'      => 'publish',
+		'order'            => $attributes['order'],
+		'orderby'          => $attributes['orderBy'],
+		'suppress_filters' => false,
 	);
 
 	if ( isset( $attributes['categories'] ) ) {
-		$args['category'] = $attributes['categories'];
+		$args['category'] = join( ',', $attributes['categories'] );
```

Comment:
`categories` is [declared in the block type as a string](https://github.com/WordPress/gutenberg/blob/171fedc2846a98b4b5dc397c265e54173ece09cf/packages/block-library/src/latest-posts/index.php#L93-L95), so `join` is probably wrong here. Did you run into non-string values for `categories`?

Reply:
sorry for delay, you're right fixed